nifi-registry-core/nifi-registry-web-ui/src/main/webpack.common.js

```diff
@@ -0,0 +1,199 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const webpack = require('webpack');
+const path = require('path');
+const FixStyleOnlyEntriesPlugin = require('webpack-fix-style-only-entries');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+const webpackAlias = require('./webpack.alias');
+
+module.exports = {
+    // Deployment target
+    target: 'web',
+
+    // Starting point of building the bundles
+    entry: {
+        // JS files
+        'nf-registry.bundle.min': path.resolve(__dirname, 'webapp/nf-registry-bootstrap.js'),
+
+        // SCSS files
+        'nf-registry.style.min': [
+            path.resolve(__dirname, 'webapp/theming/nf-registry.scss'),
+        ]
+    },
+
+    // Output bundles
+    output: {
+        // add the content hash for auto cache-busting
+        filename: '[name].[contenthash].js',
+        path: path.resolve(__dirname, './'),
+        publicPath: 'nifi-registry/'
+    },
+
+    optimization: {
+        splitChunks: {
+            cacheGroups: {
+                vendor: {
+                    chunks: 'initial',
+                    test: path.resolve(__dirname, 'node_modules'),
+                    name: 'vendor.min',
+                    enforce: true
+                }
+            }
+        }
+    },
+
+    // Change how modules are resolved
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js'],
+        alias: webpackAlias
+    },
+
+    // Polyfill or mock certain Node.js globals and modules
+    node: {
+        console: true
+    },
+
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                include: [
+                    path.resolve(__dirname, 'webapp'),
+                ],
+                use: [
+                    {
+                        loader: 'cache-loader'
+                    },
+                    {
+                        loader: 'ts-loader'
+                    }
+                ]
+            },
+            {
+                test: /\.js$/,
+                include: [
+                    path.resolve(__dirname, 'node_modules/@nifi-fds/core')
+                ],
+                use: [
+                    {
+                        loader: 'cache-loader'
+                    },
+                    {
+                        loader: path.resolve(__dirname, 'systemjs-text-to-html-loader')
+                    }
+                ]
+            },
+            {
+                test: /\.js$/,
+                include: [
+                    path.resolve(__dirname, 'webapp')
+                ],
+                use: [
+                    {
+                        loader: 'cache-loader'
+                    },
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: ['@babel/preset-env'],
+                            plugins: [
```

Comment:
I think these plugins are not used/required so they can be removed. Please make sure you also remove them from the `package.json`.